bundle/manifests/supported-nic-ids_v1_configmap.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: v1
 data:
   Broadcom_bnxt_en_BCM57414_NetXtreme-E: 14e4 16d7 16c1
-  Broadcom_bnxt_en_BCM57504_NetXtreme-E: 14e4 1751 1806
```

Comment:
this one you can add.

this file only exist on d/s

Reply:
Rhis file is autogenerated using config/manifests/bases/sriov-network-operator_configmap.yaml, so it will not be present after running "make bundle manifests"

Reply:
you are right sorry!